core/src/actors/json_rpc/json_rpc_methods.rs

```diff
@@ -0,0 +1,213 @@
+#[cfg(test)]
+use self::mock_actix::System;
+use crate::actors::blocks_manager::{messages::AddNewBlock, BlocksManager};
+#[cfg(not(test))]
+use actix::System;
+use jsonrpc_core::{IoHandler, Params, Value};
+use log::info;
+use serde_derive::{Deserialize, Serialize};
+use witnet_data_structures::chain::Block;
+
+/// Define the JSON-RPC interface:
+/// All the methods available through JSON-RPC
+pub fn jsonrpc_io_handler() -> IoHandler<()> {
+    let mut io = IoHandler::new();
+
+    io.add_method("inventory", |params: Params| inventory(params.parse()?));
+
+    io
+}
+
+/// Inventory element: block, tx, etc
+#[derive(Debug, Eq, PartialEq, Clone, Serialize, Deserialize)]
+pub enum InventoryElement {
```

Comment:
We are calling this `InventoryItem` pretty much everywhere else.
```suggestion
pub enum InventoryItem {
```